metrics/cli/docs/features/doctor_output_improvements/diagrams/ci_integrations_validator_class_diagram.puml

```diff
@@ -0,0 +1,112 @@
+@startuml ci_integrations_validator_class_diagram
+
+package validation.lib.src.validation {
+    package model {
+        package builder {
+            class ValidationResultBuilder {
+                - _results: UnmodifiableMapView<ValidationTarget, TargetValidationResult>
```

Comment:
🔨 
```suggestion
                - _results: Map<ValidationTarget, TargetValidationResult>
```
It seems like we have a modifiable `Map` in the `ValidationResultBuilder`.